poly/helper.go

```diff
@@ -0,0 +1,107 @@
+package poly
```

Comment:
Is this code mostly or only used by _test.go test-suite code?  If so, it should also have a _test.go name, so that it doesn't get compiled into the non-test "deployment" code-base.  For example, 'help_test.go'?